trento/xml/article_sap_trento.xml

```diff
@@ -669,6 +669,124 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
+      </listitem>
+      <listitem>
+        <para>SAP administrator with Trento operation access: users with
+          all:check_execution and cleanup:all permissions.</para>
+      </listitem>
+    </itemizedlist>
+    <para>The default admin user created during the installation process is
+      granted all:all permissions and cannot be modified or deleted. It should
+      only be used to create a first user manager. That is, a user with
+      all:users permissions who in turn will create all the other required
+      users. Once a user with all:users permissions is created, the default
+      admin user should be regarded as a fall back, emergency user to be used
+      only in case all other access to the console is lost. Should the password
+      of the default admin user itself be lost, it can be reset by updating the
+      helm chart or the web component configuration, depending on which
+      deployment method was used to install Trento Server. </para>
+    <para>User passwords, including the default admin user password, must comply
+      with the following rules:</para>
+    <itemizedlist>
+      <listitem>
+        <para>They have at least 8 characters</para>
+      </listitem>
+      <listitem>
+        <para>They do not have three consecutive numbers or letters (for
+          example: 111 or aaa)</para>
+      </listitem>
+      <listitem>
+        <para>They do not have four consecutive numbers or letters (for example:
+          1234, abcd or ABCD)</para>
+      </listitem>
+    </itemizedlist>
+    <para>The Create User and Edit User views provide a built-in generation
+      password action button that allows user managers to easily generate
+      secure, compliant passwords. The user manager should provide the user with
+      their password using a secure, authorized channel. </para>
+    <para>Users can reset their passwords in their profile views, where they can
+      also update their Full Names and their email addresses as well as activate
+      Multi Factor Authentication using an Authenticator app of their choice.
+      Multi-factor authentication increases the security of a user account by
+      requesting a temporary second password or code when logging in the
+      console. User managers can disable Multi-Factor authentication for any
+      given user that has it enabled, but cannot enable it on their behalf. The
+      default admin user cannot enable Multi-Factor Authentication for
+      itself.</para>
+    <note>
+      <title>Security Tip for Multi-Factor Authentication</title>
+      <para>Since Multi-Factor Authentication cannot be enabled for
+      the default admin user, keeping its password safe is essential. If the
+      default admin user password is compromised, reset it immediately by
+      updating the helm chart or the web component configuration, depending on
+      which deployment method was used to install &t.server;. </para>
+    </note>
```

Comment:
```suggestion
      <para>Since multi-factor authentication cannot be enabled for
      the default admin user, keeping its password safe is imperative. If the
      default admin user's password is compromised, reset it immediately by
      updating the helm chart or the web component configuration, depending on
      which deployment method was used to install &t.server;. </para>
    </note>
```